sync-wp/components/config/screen.py

```
@@ -0,0 +1,105 @@
+from PySide6.QtWidgets import (QWidget, QVBoxLayout,
+                              QFormLayout, QGroupBox, QLabel, 
+                              QLineEdit, QPushButton, QMessageBox)
+from config import SITE_URL
+from .config_manager import ConfigManager
+
+class ConfigScreen(QWidget):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)